Brightspace Extensions/BrightspaceExtensions.postman_collection.json

```diff
@@ -0,0 +1,385 @@
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"ExtensionId\": {{Extension1}},\r\n    \"GlobalId\": null,\r\n    \"Name\": \"THIS HAS BEEN CHANGED\",\r\n    \"IsLocal\": true,\r\n    \"ShortDescription\": \"bob1\",\r\n    \"State\": 0,\r\n    \"Website\": \"bob\",\r\n    \"Email\": \"bob@bob\",\r\n    \"SupportSite\": \"bob.com\",\r\n    \"CreatedDate\": \"2024-05-08T13:51:59.117Z\",\r\n    \"CreatedBy\": 9,\r\n    \"LastUpdatedDate\": \"2024-05-08T13:51:59.117Z\",\r\n    \"LastUpdatedBy\": 9,\r\n    \"LastPublishedDate\": null,\r\n    \"LastPublishedBy\": \"9\",\r\n    \"Tags\": \"b\",\r\n    \"Categories\": \"bobFTW\"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension1}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension1}}"
+							]
+						},
+						"description": "Update Extension 1's Name"
+					},
+					"response": []
+				},
+				{
+					"name": "Get Extension 1",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);\r",
+									"\r",
+									"pm.test(\"Checking that extension one's name has been changed\", function() { jsonData.Name == \"THIS HAS BEEN CHANGED\" })\r",
+									"\r",
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension1}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension1}}"
+							]
+						},
+						"description": "Ensure Extension 1's name has changed as expected"
+					},
+					"response": []
+				},
+				{
+					"name": "Get All Extensions",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);\r",
+									"var initialExtnesionCount = pm.collectionVariables.get(\"ExtensionNumber\");\r",
+									"pm.test(\"Checking that extension one's name has been changed\", function() { jsonData.length == initialExtnesionCount+2 })\r",
```

Comment:
This test description could be updated to `Checking that 2 extensions have been created`.